README.md

```diff
@@ -1,25 +1,17 @@
 # OpenJDK Mirror Scripts
 
-These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are responsible for updating the AdoptOpenJDK clones of the various OpenJDK mecurial forests that we are interested in building.
+These scripts are run at https://ci.adoptopenjdk.net/view/git-mirrors/ and are responsible for updating the Eclipse Adoptium clones of the various OpenJDK Skara github repositories that we are interested in building.
 
 ## For developers
 
-OpenJDK Source Control repositories are managed using Mercurial. The scripts within this project mirror those Mercurial repositories
-to github.com/adoptium/jdk\<NN\>u , which then serves as a mirror for Adoptium to build the Hotspot variants, as well as providing branches for adding any Adoptium vendor "patches" needed.
+OpenJDK Source Control repositories are now managed using the GitHub (project Skara). The Git repositories are mirrored at OpenJDK.
 
-The OpenJDK Mercurial repositories fall under three categories:
-```
-- JDK8 Update Repositories   : hg.openjdk.java.net/jdk8u/jdk8u/(corba|hotspot|jaxp|jaxws|nashorn|jdk|langtools)
-- JDK11+ Update Repositories : hg.openjdk.java.net/jdk-updates/jdk<NN>u
-- JDK "next" Repository      : hg.openjdk.java.net/jdk/jdk
-```
+Examples of converted repositories are available at https://github.com/openjdk/.
 
-# JDK8 Update Repositories
-jdk8u/... scripts
+# Skara repos and processes
 
-TBD
+Historical information about the migration to OpenJDK GitHub from Mercurial is available at https://openjdk.org/jeps/369
 
-# JDK11+ Update, JDK "next" Repositories
 The **mercurialToGit.sh** script mirrors and merges the corresponding Mercurial repository into the matching Adoptium mirror:
```

Comment:
Sorry I wasn't clear, source lines 15-21 need deleting, ie.the Mercurial bits:
```
The **mercurialToGit.sh** script mirrors and merges the corresponding Mercurial repository into the matching Adoptium mirror:
```
hg.openjdk.java.net/jdk-updates/jdk<NN>u     --->    github.com/adoptium/jdk<NN>u (master)
hg.openjdk.java.net/jdk/jdk                  --->    github.com/adoptium/jdk(master)
```
This mirroring utilizes the "git-remote-hg" Mercurial fast importer plugin (https://github.com/felipec/git-remote-hg), and mirrors
the Mercurial "default" branch to the "master" branch in the git repo.
```